product_mandatory_products/views/product_template_view.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+
+    <record id="mandatory_product_ids_product_template_form_view" model="ir.ui.view">
+        <field name="name">mandatory.product.ids.template.product.form</field>
+        <field name="model">product.template</field>
+        <field name="inherit_id" ref="product.product_template_form_view" />
+        <field name="arch" type="xml">
+            <xpath expr="//group[@name='upsell']" position="inside">
+                <field
+          name="mandatory_product_ids"
+          widget="many2many_tags"
+          domain="[('id', '!=', active_id)]"
```

Comment:
I am not sure what the intended purpose of that domain is. From my understanding, it should lead to only relevant product_ids be displayed. But aren't things with active = False not hidden by Odoo anyways, normally?